core/src/main.rs

```diff
@@ -77,17 +79,20 @@ static ORACLE_CORE_ASCII: &str = r#"
 #[derive(Parser)]
 #[clap(author, version, about, long_about = None)]
 struct Args {
-    #[clap(long = "Run oracle core in read-only mode")]
-    read_only: bool,
+    #[clap(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand)]
+enum Command {
+    Bootstrap { yaml_config_name: String },
+    Run,
```

Comment:
The commands it will look like `oracle-core bootstrap bootstrap_config.yaml` and `oracle-core run`, like how it is with `git`.

Reply:
Sorry, what I meant with git is that it has the form `git <command>`, where `command` could be `pull/push/add/rebase` etc. You asked for `--bootstrap bootstrap_config.yaml` in the issue, so instead of the `--bootstrap` flag, it follows the form of a command `oracle-core bootstrap bootstrap_config.yaml` instead.

Now I think we need the `run` command (which will simply run the main loop) because `oracle-core` as a CLI app needs to distinguish from the other commands like `extract-reward-tokens`, `print-reward-tokens` etc. Am I making sense?

Reply:
Oh, I see. Yes, it makes sense. Initially, I thought that the main loop will be run if no commands passed, but I like separate command(run) better now.